docs/io/unified_table_fits.rst

```diff
@@ -0,0 +1,625 @@
+described in :ref:`fits_time_column`. Once it has been identified, its datatype
+is checked in order to determine its representation format. Since ISO 8601
+``datetime`` format is the only string representation of time, a time
+coordinate column having string datatype will be automatically read as a
+`~astropy.time.Time` object with ``format='fits'`` ('fits' represents the FITS
+ISO 8601 format).
+
+As this format does not imply a particular timescale, it is determined using
+the timescale keywords in the header (``TCTYP`` or ``TIMESYS``) or their
+defaults. The other time coordinate information is also determined in the same
+way, using the time coordinate frame keywords. All ISO 8601 times are relative
+to a globally accepted zero point (year 0 corresponds to 1 BCE) and are thus
+not relative to the reference time keywords (MJDREF, JDREF, or DATEREF).
+Hence, these keywords will be ignored while dealing with ISO 8601 time columns.
+
+.. note::
+
+   Reading FITS files with time coordinate columns *may* fail. ``astropy``
+   supports a large subset of these files, but there are still some FITS files
+   which are not compliant with any aspect of the standard. If you have such a
+   file, please do not hesitate to let us know (by opening an issue in the
+   `issue tracker <https://github.com/astropy/astropy/issues>`_).
+
+   Also, reading a column having ``TTYPEn = ‘TIME’`` as `~astropy.time.Time`
+   will fail if ``TUNITn`` for the column is not a FITS-recognized time unit.
+
+.. _unified_table_fits_details:
+
+Details
+~~~~~~~
+
+Time as a dimension in astronomical data presents challenges in its
+representation in FITS files. The standard has therefore been extended to
+describe rigorously the time coordinate in the ``World Coordinate System``
+framework. Refer to `FITS WCS paper IV
+<https://ui.adsabs.harvard.edu/abs/2015A%26A...574A..36R/>`_ for details.
+
+Allowing ``Time`` columns to be written as time coordinate
+columns in FITS tables thus involves storing time values in a way that
+ensures retention of precision and mapping the associated metadata to the
+relevant FITS keywords.
+
+In accordance with the standard, which states that in binary tables one may use
+pairs of doubles, the ``astropy`` Time column is written in such a table as a
+vector of two doubles ``(TFORMn = ‘2D’) (jd1, jd2)`` where ``JD = jd1 + jd2``.
+This reproduces the time values to double-double precision and is the
+"lossless" version, exploiting the higher precision provided in binary tables.
+Note that ``jd1`` is always a half-integer or integer, while ``abs(jd2) < 1``.
+"Round-tripping" of ``astropy``-written FITS binary tables containing time
+coordinate columns has been partially achieved by mapping selected metadata,
+``scale`` and singular ``location`` of `~astropy.time.Time`, to corresponding
+keywords. Note that the arbitrary metadata allowed in `~astropy.table.Table`
+objects within the ``meta`` dict is not written and will be lost.
```

Comment:
I first read this as "arbitrary values", but what you mean is "arbitrary datatypes". If my meta has strings, they will turn up as FITS header cards. Not sure how to say that here to preserve the meaning. "Note the metadata in ... that is not just a simple string is not written an will be lost"?

Reply:
That sentence about Table meta representation in FITS was completely out of place here, so I just removed it. Sometimes less is more.